pandas/tests/io/test_html.py

```diff
@@ -1286,3 +1286,65 @@ def test_parse_path_object(self, datapath):
         df1 = self.read_html(file_path_string)[0]
         df2 = self.read_html(file_path)[0]
         tm.assert_frame_equal(df1, df2)
+
+    def test_extract_hrefs_body(self):
+        # GH 13141:
+        # read_html argument to interpret hyperlinks as links (not merely text)
+        result = self.read_html(
+            """
+          <table>
+            <tr>
+              <th>HTTP</th>
+              <th>FTP</th>
+              <th><a href="https://en.wiktionary.org/wiki/linkless">None</a></th>
+            </tr>
+            <tr>
+              <td><a href="https://en.wikipedia.org/">Wikipedia</a></td>
+              <td><a href="ftp://ftp.us.debian.org/">Debian</a></td>
+              <td>Linkless</td>
+            </tr>
+          </table>
+          """,
+            extract_hrefs="body",
+        )[0]
+
+        expected = DataFrame(
+            [
+                [
+                    ("Wikipedia", "https://en.wikipedia.org/"),
+                    ("Debian", "ftp://ftp.us.debian.org/"),
+                    ("Linkless",),
+                ]
+            ],
+            columns=(
+                "HTTP",
+                "FTP",
+                "None",
+            ),
+        )
+
+        tm.assert_frame_equal(result, expected)
+
+    def test_extract_hrefs_header(self):
```

Comment:
need a test that validatees the extract_href arg is among the required values or raises.

Reply:
Required values?